bin/rtlcss.js

```diff
@@ -180,11 +188,11 @@ if (!shouldBreak) {
       }
       var i = 0
       ;(function next () {
-        var file = list[i++]
```

Comment:
No need to introduce a new variable, you can get the file name using [path.basename](https://nodejs.org/api/path.html#path_path_basename_p_ext).